openmetadata-ui/src/main/resources/ui/src/locale/languages/de-de.json

```diff
@@ -567,6 +567,7 @@
     "glossary-term-lowercase": "glossarbegriff",
     "glossary-term-lowercase-plural": "glossarbegriffe",
     "glossary-term-plural": "Glossarbegriffe",
+    "glossary-version": "Glossary version",
```

Comment:
would prefer to have a common key for this

`"entity-version": "{{entity}} version",`